### ISSUE TITLE
RailsガイドとRailsGuidesを比較して英語を勉強する何か

### ISSUE THREAD

**Issue by yasulab** [yasslab/railsguides.jp#212]
<img width="854" alt="screen shot 2015-09-16 at 15 21 58" src="https://cloud.githubusercontent.com/assets/155807/9897725/bec6c126-5c86-11e5-9573-c47ab4a9154a.png">

次の記事を読んでいたら、「Railsガイドの原文を読みながら、辛くなったら訳文を参照できるというRubyistのための英語学習ガイド」という謎アイデアが舞い降りてきたので、衝動でIssueに挙げておきます :stars: 

[英語力をアップさせる知見がいっぱい！「Rubyistのための英語勉強会」を開催しました](http://blog.jnito.com/entry/2015/09/02/120620)

cf. [Railsをはじめよう](http://railsguides.jp/getting_started.html) / [Getting Started with Rails](http://guides.rubyonrails.org/getting_started.html)

**Comment by yui-knk** [comment]
@yasulab  Railsをはじめよう のlinkに"%08"が入っていたので修正しました

**Comment by yasulab** [comment]
@yui-knk おぉ、訂正ありがとうございます :wrench: :star2:

**Comment by hachi8833** [comment]
これ面白いですね！

JSでツールチップ的に訳文を表示できるアドインになったりするといいですね。

**Comment by yasulab** [comment]
> JSでツールチップ的に訳文を表示できるアドインになったりするといいですね。

そうですね :smile_cat: 突発的なアイデアではありますが、もしやるなら僕もこんなイメージですね :speech_balloon:

**Comment by ymoto** [comment]
上記スライドの作者です。このような形で拾っていただいて、感謝です。

JSでツールチップ的に訳文を表示というアイデアなら、ご存じかもしれませんが下記が参考になるかと思います。

http://melborne.github.io/2014/02/17/update-togglate-for-renewed-proposal-to-translation/
